tests/unit/snow/test_dialect_utils.py

```diff
@@ -119,3 +119,18 @@ def test_query_with_error():
 
     error = check_for_unsupported_lca(dialect, sql, filename)
     assert not error
+
+
+def test_query_with_same_alias_and_column_name():
+    dialect = "SNOWFLAKE"
+    sql = """
+    select ca_zip
```

Comment:
can you add test case where there are more than one column as well as for window expression

Reply:
Added the window expression in test